### ISSUE TITLE
Allow multiline field value for JSON request body

### ISSUE THREAD

**Issue by JokerQyou** [ducaale/xh#45]
I don't know if it's supported by `httpie`, but I think it's a valid use case. Basically when requesting with JSON body, one do it like this:

```bash
ht -j -v put https://httpbin.org/put SingleLine=my_string
PUT /put HTTP/1.1
accept: application/json, */*
accept-encoding: gzip, deflate
connection: keep-alive
content-length: 26
content-type: application/json
host: httpbin.org

{
    "SingleLine": "my_string"
}

HTTP/2.0 200 OK
access-control-allow-credentials: true
access-control-allow-origin: *
content-length: 474
content-type: application/json
date: Tue, 09 Feb 2021 06:53:46 GMT
server: gunicorn/19.9.0

{
    "args": {},
    "data": "{\"SingleLine\":\"my_string\"}",
    "files": {},
    "form": {},
    "headers": {
        "Accept": "application/json, */*",
        "Accept-Encoding": "gzip, deflate",
        "Content-Length": "26",
        "Content-Type": "application/json",
        "Host": "httpbin.org",
        "X-Amzn-Trace-Id": "Root=1-6022317a-<removed>"
    },
    "json": {
        "SingleLine": "my_string"
    },
    "origin": "<removed>",
    "url": "https://httpbin.org/put"
}
```

That worked flawlessly, and I really like the way JSON body was constructed by using separated key-value pairs, it's much easy to write, and easier to type without  typo.

But when one of the values contains multiple lines, `ht` is not happy:

```bash
ht -j -v put https://httpbin.org/put SingleLine=my_string MultiLine=my\nstring
error: Invalid value for '<REQUEST_ITEM>...': error: "MultiLine=my\nstring" is not a valid value

```

This happens with `ht 0.5.0` which is the latest version. I know for `httpie` you can pipe raw JSON data into it so any form of data is supported, but simple key-value construction is just so much better. Will `ht` support this?

**Comment by ducaale** [comment]
Could write down which OS and shell you were using when you encountered this bug? I have tested your command on both Powershell, Bash, and Zsh but couldn't manage to reproduce the issue.

**Comment by blyxxyz** [comment]
It was probably fish. Seems like it handles newlines slightly differently, I use it too so I didn't notice.

This version works on bash:
```sh
ht -j -v put https://httpbin.org/put SingleLine=my_string $'MultiLine=my\nstring'
```

**Comment by JokerQyou** [comment]
Yep it's `fish` shell. Sorry I forgot to mention it in my initial comment.

**Comment by ducaale** [comment]
This is now fixed as part of v0.7.0 release.
